### PR TITLE
Refactor employee stress

### LINKED ISSUE #143: Limit employee stress with upper and lower bounds

### PR DESCRIPTION
**Pull request by mertwole**
Resolves #143

### CODE REVIEW

File: JamGame/Assets/Scripts/Employee/EmployeeImpl.cs [hunk 1]
```diff
@@ -125,7 +126,7 @@ NotifyCollectionChangedEventArgs original_args
         private void Start()
         {
             controller = GetComponent<ControllerImpl>();
-            Stress = GetComponent<StressMeter>();
+            Stress = GetComponent<StressMeterImpl>();
```
Comment (Kiuh) [inline JamGame/Assets/Scripts/Employee/EmployeeImpl.cs:129]: Have you missed `[RequireComponent(typeof(StressMeterImpl))]`?

File: JamGame/Assets/Scripts/Employee/StressMeter/Inspector.cs [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+using Employee;
+using Employee.StressMeter;
+using UnityEditor;
+using UnityEngine;
+
+namespace Assets.Scripts.Employee.StressMeter
+{
+    [CustomEditor(typeof(StressMeterImpl))]
+    [CanEditMultipleObjects]
+    public class Inspector : Editor
```
Comment (Kiuh) [inline JamGame/Assets/Scripts/Employee/StressMeter/Inspector.cs:11]: Custom editor don't work with Odin Inspector, so make with Odin, partial class with bunch of attributes or create custom [value drawer](https://odininspector.com/tutorials/how-to-create-custom-drawers-using-odin/how-to-make-a-healthbar-attribute) for example. Also many of things you can do with groups attributes for layouting.

Comment (Kiuh) [reply JamGame/Assets/Scripts/Employee/StressMeter/Inspector.cs:11]: Also, when `StressMeterImpl` re-drawed in editor, prefab always saves himself.

**Review by Kiuh — CHANGES_REQUESTED** [review]

File: JamGame/Assets/Scripts/Employee/StressMeter/Inspector.cs (hunk 2 above)
Comment (mertwole) [reply JamGame/Assets/Scripts/Employee/StressMeter/Inspector.cs:11]: Added check to not make object dirty each frame

Comment (mertwole) [reply JamGame/Assets/Scripts/Employee/StressMeter/Inspector.cs:11]: Discussed

**Pull request merged**